uk.co.compucorp.civicrm.tasksassignments/css/civitasks.css

```diff
@@ -11703,43 +11703,17 @@ fieldset[disabled]
 }
 
 .ct-spinner-cover {
-  -moz-border-radius: 0;
-  -webkit-border-radius: 0;
-  border-radius: 0;
-  background-color: rgba(255, 255, 255, 0.5) !important;
   position: absolute;
-  width: 100%;
-  height: 100%;
-  top: 0;
-  left: 0;
+  top: 50%;
```

Comment:
Not sure it is centered. Same issue as in the https://github.com/civicrm/civihr/pull/2333

Reply:
Well, tours out top: 50%; left:50% is not enough for positioned element to keep in center. So, I have created a fiddle http://jsfiddle.net/6xo11zwv/266/ with transform: translate(-50%, -50%); as well. I have checked with different browsers and it works. I have update the the style.